subjects/ascii-art/README.md

```diff
@@ -32,10 +32,10 @@ $$@@$$$$$$$$``````````@@$$@@$$$$$$
 ### Instructions
 
 - Your project must be written in **Go**.
-- The code must respect the [**good practices**](https://public.01-edu.org/subjects/good-practices/).
+- The code must respect the [**good practices**](https://[[DOMAIN]]/root/public/src/branch/master/subjects/good-practices/README.md).
```

Comment:
This link doesn't work, does it ?
The syntax is now `((DOMAIN))`
You can test it if you go to https://git.dev.01-edu.org/root/public/src/branch/project-reviews/subjects/ascii-art

Reply:
Your local `public` repository updates every 10 minutes, you can sync manually by going to https://git.dev.01-edu.org/root/public/settings and clicking on "Synchronize Now"